modules/recon/hosts-hosts/ipstack.py

```diff
@@ -22,8 +22,8 @@ def module_run(self, hosts):
             except ValueError:
                 self.error(f"Invalid JSON response for '{host}'.\n{resp.text}")
                 continue
-            region = ', '.join([str(jsonobj[x]).title() for x in ['city', 'region_name'] if jsonobj[x]]) or None
-            country = jsonobj['country_name'].title()
+            region = ', '.join([str(jsonobj[x]) for x in ['city', 'region_name'] if jsonobj[x]]) or None
+            country = str(jsonobj['country_name'])
```

Comment:
Curious as to why you needed to add `str(` and why you removed the `title` casing?

Reply:
So I removed .title() because when you get a None back from the API such as {'country_name': None} you would error out. As for the str after I submitted this pull request I think those should be removed even from LAT And LONG because if you keep them there it fills the DB with None when it gets that back from the API. I just did some testing with that removed it works just fine and doesn't fill the DB entry up with crap. Up to you, I can update this one without the str if you think that is the functionality you wanted.

Reply:
Yeah. Let's do that. Thanks!

Reply:
Pushed